docs/usage/migrating-existing-etcd-cluster.md

```diff
@@ -0,0 +1,159 @@
+# Migrating an Existing etcd Cluster to an etcd-druid Managed etcd Cluster
+
+This guide provides a comprehensive procedure to migrate an existing (unmanaged) etcd cluster into an etcd-druid managed etcd cluster.
+
+---
+
+## Prerequisites
+
+- **Deploy etcd-druid:** Follow the installation steps in the [etcd-druid documentation](https://gardener.github.io/etcd-druid/deployment/getting-started-locally/getting-started-locally.html) to deploy the etcd-druid operator in your environment.
+
+## Overview
+
+`etcd-druid` supports the integration of new etcd members into an existing `etcd` cluster through the `etcd.spec.etcd.existingCluster` configuration. This capability ensures that when new managed members are added, they automatically join the pre-existing cluster, facilitating seamless migrations.
+
+> **Note:**  
+> - The `existingCluster` configuration is evaluated during the bootstrapping of new cluster members.  
+> - It is crucial to utilize identical TLS certificates for the new etcd Custom Resource (CR) when joining an existing cluster to ensure secure communication.
+
+## Key Considerations
+
+### Single-Node `etcd` Cluster Migration
+
+- **From Single-Node to Three-Node:** Transitioning from a single-node to a three-node etcd cluster can be achieved with zero downtime. However, once expanded to a multi-member configuration, reducing back to a single-node setup is not supported by `etcd-druid`.
+- **From Single-Node to Single-Node:** Migrations of this nature typically involve some downtime. You are required to capture a snapshot using the [etcd-backup-restore tool](https://github.com/gardener/etcd-backup-restore/blob/master/docs/deployment/getting_started.md#taking-scheduled-snapshot), upload it to your designated backup bucket, and configure your new etcd CR to restore from this snapshot.
```

Comment:
Why would there be downtime involved? The cluster gets scaled up from 1 to 2 members, and as long as both members are healthy, there's quorum. And then the user removes the old member from the cluster via `etcdctl member remove` call, which scales down the cluster size to 1 member, which still ensures quorum. I don't see why this would 'typically involve some downtime". Yes, there is a possibility of quorum loss if one of the members becomes unhealthy during this migration process. But that's a possibility for a 3-node cluster migration too, if 3 members go down.

Can you please re-word this?